### DISCUSSION TITLE
Authenticator always failed

### DISCUSSION THREAD

**Discussion by josefhs11** [gravitational/teleport#12518] [Q&A]
![teleport-error](https://user-images.githubusercontent.com/73809934/167373597-b693ce6f-824e-4cd8-86c0-99906615e77a.png)

Install teleport for the first time and got this error.

Kindly help me with the solution.

**Comment by stevenGravy** [comment]
Please provide how you configured it @josefhs11  in your /etc/teleport.yaml.  If you are attempting to use the acme setting for retrieving certs the letencrypt service needs to be able to invoke your FQDN through port 443.

**Reply by josefhs11** [reply]
Hi @stevenGravy 
It's just the default configuration, generated using  `teleport configure > teleport.yaml`  command
![tele](https://user-images.githubusercontent.com/73809934/168017807-b1fc0f25-5359-4099-8094-78253370e5ad.png)

**Comment by webvictim** [comment]
What command are you running when you get that error?

**Reply by josefhs11** [reply]
I start the teleport with this command
teleport start -c /etc/teleport/teleport.yaml
